scripts/serve_dist.sh

```diff
@@ -1,18 +1,36 @@
 #!/usr/bin/env bash
 
-set -e 
+set -e
 set -u
 set -o pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+protocol="https"
+
+if [ $# -eq 1 ]
+  then
+    protocol="http"
```

Comment:
this seems a little funky for handling arguments, if you want to avoid having to parse arguments properly could also do something like

```
protocol="${PROTOCOL:-https}"
```
If somebody exports PROTOCOL before calling this it would use that, `https` being default. Cleaner would be to do proper arg parsing but get that can be overkill since CI is probably the only one using this